Level 01 (Spaghetti Code)/app.py

```diff
@@ -20,8 +20,9 @@ def hello_world():
 
 @app.route("/users", methods=["GET"])
 def get_users():
-    """Users route. method = GET, this route will return all the user in existence"""
-    # Connect to mysql Database
+    """Users route. method = GET, this route will return all the user in existence
+    Connect to mysql Database"""
```

Comment:
```suggestion
    """Users route. method = GET, this route will return all the user in existence"""
    
    # Connect to mysql Database
```
El comentario de la conexión no tiene nada que ver con el docstring